web/hooks/use-mod-reports.ts

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react'
+import { api } from 'web/lib/firebase/api'
+import { ModReport, ReportStatus } from 'common/mod-report'
+import { keyBy, mapValues } from 'lodash'
+
+export const useModReports = () => {
+  const [reports, setReports] = useState<ModReport[] | undefined>(undefined)
+  const [reportStatuses, setReportStatuses] = useState<{
+    [key: number]: ReportStatus
+  }>({})
+  const [modNotes, setModNotes] = useState<{
+    [key: number]: string | undefined
+  }>({})
+
+  const getModReports = async () => {
+    try {
+      const response = await api('get-mod-reports', {})
+      if (response && response.status === 'success') {
+        const newReports = response.reports
+
+        const sortedReports = newReports
+          .filter((report: ModReport) => report.created_time)
+          .sort(
+            (a: ModReport, b: ModReport) =>
+              new Date(b.created_time ?? 0).getTime() -
+              new Date(a.created_time ?? 0).getTime()
```

Comment:
I thought all reports would have a `created_time` making the filter unnecessary (and `?? 0`)